aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ConfigEvent.java

```diff
@@ -13,400 +13,84 @@
-        setInvokingEvent(invokingEvent);
-        return this;
-    }
-    
-    /**
-     * Gets the JSON-encoded map containing the AWS Config rule parameters.
-     * 
-     */
-    public String getRuleParameters() {
-        return ruleParameters;
-    }
-
-    /**
-     * Sets the JSON-encoded map containing the AWS Config rule parameters.
-     * @param ruleParameters String containing the AWS Config rule parameters.
-     */
-    public void setRuleParameters(String ruleParameters) {
-        this.ruleParameters = ruleParameters;
-    }
-
-    /**
-     * @param ruleParameters String with rule parameters
-     * @return ConfigEvent
-     */
-    public ConfigEvent withRuleParameters(String ruleParameters) {
-        setRuleParameters(ruleParameters);
-        return this;
-    }
-    
-    /**
-     * Gets the token associated with the invocation of the AWS Config rule's Lambda function.
-     * 
-     */
-    public String getResultToken() {
-        return resultToken;
-    }
-
-    /**
-     * Sets the token associated with the invocation of the AWS Config rule's Lambda function.
-     * @param resultToken String containing the token associated to the invocation.
-     */
-    public void setResultToken(String resultToken) {
-        this.resultToken = resultToken;
-    }
-
-    /**
-     * @param resultToken result token
-     * @return ConfigEvent
-     */
-    public ConfigEvent withResultToken(String resultToken) {
-        setResultToken(resultToken);
-        return this;
-    }
-    
-    /**
-     * Gets the ARN of the AWS Config rule that triggered the event.
-     * 
-     */
-    public String getConfigRuleArn() {
-        return configRuleArn;
-    }
-
     /**
-     * Sets the ARN of the AWS Config rule that triggered the event.
-     * @param configRuleArn String containing the AWS Config rule ARN.
+     * The AWS Config event version.
+     *
      */
-    public void setConfigRuleArn(String configRuleArn) {
-        this.configRuleArn = configRuleArn;
-    }
+    private String version;
 
     /**
-     * @param configRuleArn config rule for arn
-     * @return ConfigEvent
-     */
-    public ConfigEvent withConfigRuleArn(String configRuleArn) {
-        setConfigRuleArn(configRuleArn);
-        return this;
-    }
-    
-    /**
-     * Gets the ID of the AWS Config rule that triggered the event.
-     * 
+     * The JSON-encoded notification published by AWS Config.
+     *
      */
-    public String getConfigRuleId() {
-        return configRuleId;
-    }
+    private String invokingEvent;
 
     /**
-     * Sets the ID of the AWS Config rule that triggered the event.
-     * @param configRuleId String containing the AWS Config rule ID.
+     * The JSON-encoded map containing the AWS Config rule parameters.
+     *
      */
-    public void setConfigRuleId(String configRuleId) {
-        this.configRuleId = configRuleId;
-    }
+    private String ruleParameters;
```

Comment:
Because `ruleParameters` is a String. 

Ref: https://docs.aws.amazon.com/config/latest/developerguide/evaluate-config_develop-rules_example-events.html